ca.erl

```diff
@@ -1,16 +1,16 @@
 -module(ca).
--export([process_state/2]).
--export_type([state/0, rule/0]).
+-export([process_state/2, apply_rules/2, make_state/2, get_start/0, render/1, render_cell/1]).
```

Comment:
Line 2 is too long: -export([process_state/2, apply_rules/2, make_state/2, get_start/0, render/1, render_cell/1])..